code/modules/reagents/chemistry/reagents/medicine.dm

```diff
@@ -1184,6 +1199,45 @@
 /datum/reagent/medicine/stimulative_agent/changeling/on_mob_delete(mob/living/L)
 	return
 
+// IPC Stimulative Agent, for the Supercharge Biochip.
+/datum/reagent/medicine/stimulative_agent/surge_plus
+	name = "Surge Plus"
+	id = "surge_plus"
+	description = "A high quality, low-viscocity gel that both supercharges processors and massively increases the efficincy of synthetic locomotive systems, allowing the user to run faster whilst also clearing stuns. \
+	If overdosed, it will cause short-circuits that will cause damage and reduce locomotive efficiancy gains."
+	reagent_state = LIQUID
+	color = "#28b581"
+	process_flags = SYNTHETIC
+	taste_description = "silicon"
+
+/datum/reagent/surge_plus/on_mob_life(mob/living/M)
+	var/update_flags = STATUS_UPDATE_NONE
+	M.AdjustParalysis(-6 SECONDS)
+	M.AdjustStunned(-6 SECONDS)
+	M.AdjustWeakened(-6 SECONDS)
+	M.AdjustKnockDown(-6 SECONDS)
+	// The 4 below benefits are from Synaptizine, because we don't need a synthetic version of that too.	
```

Comment:
synap has different metabolic rate than stim agent so this isnt technically the same as human adrenals

Reply:
I have made an IPC version of the chem because of this.